resources/deployments/mongo/deployments_external_test.go

```diff
@@ -549,6 +549,60 @@ func TestDeploymentStorageUpdateStats(t *testing.T) {
 			OutputError: ErrStorageInvalidID,
 			OutputStats: nil,
 		},
+		"no old state": {
+			InputID: "a108ae14-bb4e-455f-9b40-2ef4bab97bb7",
+			InputDeployment: &deployments.Deployment{
+				Id: StringToPointer("a108ae14-bb4e-455f-9b40-2ef4bab97bb7"),
+				Stats: map[string]int{
```

Comment:
This map is repeated 3 times here. Maybe we can have `s := map ...` and use this instead.